### PR TITLE
Remove config.json from theme

### LINKED ISSUE #10135: DOC: https://qooxdoo.org/documentation/6.0/#/desktop/gui/theming?id=theming
Fix hint to config.json

### PR DESCRIPTION
**Pull request by voger**
Please take a look if I got the changes right. I tested them in a test project and they work.

Fixes #10135

### CODE REVIEW

**Review by hkollmann — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Review by johnspackman — APPROVED** [review]

**Review by cboulanger — APPROVED** [review]
Thanks, much appreciated. Would you also update the current (6.0.0) documentation? https://github.com/qooxdoo/documentation/tree/gh-pages/6.0/desktop/gui . Thank you!

File: docs/desktop/gui/theming.md [hunk 1]
```diff
@@ -307,9 +305,9 @@ This example uses the (Monserrat font)[https://fonts.google.com/specimen/Montser
 
 Note that things like "family" are specified more than once and in different ways - the *top* level of properties (eg `size`, `family`, `bold`, etc relate to properties in the [qx.bom.webfonts.WebFont](apps://apiviewer/#qx.bom.webfonts.WebFont) class, whereas the properties of the `sources` key are slightly different.
 
-The top level properties are typically for defining what properties you would allow the browser to apply, and are analogous to the CSS properties.  For example, you can see that `family` is an array, because just like CSS the browser is to try and the first listed font family and then fallback to the second etc.
+The top level properties are typically for defining what properties you would allow the browser to apply, and are analogous to the CSS properties. For example, you can see that `family` is an array, because just like CSS the browser will try and the first listed font family and then fallback to the second etc.
```
Comment (derrell) [inline docs/desktop/gui/theming.md:308]: "will try the first" (vs "will try **and** the first")

**Review by derrell — CHANGES_REQUESTED** [review]

File: docs/desktop/gui/theming.md (hunk 1 above)
Comment (voger) [reply docs/desktop/gui/theming.md]: Done

**Comment by voger** [comment]
Changed to draft because there are some other changes that may be added. Those changes are submitted to the V6.0 documentation and once they are merged there I will submit them here too.

**Review by johnspackman — APPROVED** [review]

**Comment by voger** [comment]
Added changes made to the V6.0 documentation. Now both files are the same.

File: docs/desktop/gui/theming.md (hunk 1 above)
Comment (cboulanger) [reply docs/desktop/gui/theming.md]: @derrell can you have a look again and approve?

**Review by derrell — APPROVED** [review]

**Pull request merged**